pyhmy/exceptions.py

```diff
@@ -12,3 +12,20 @@ class InvalidRPCReplyError(RuntimeError):
 
     def __init__(self, method, endpoint):
         super().__init__(f'Unexpected reply for {method} from {endpoint}')
+
+class InvalidValidatorError(ValueError):
```

Comment:
Add a `__repr__` or `__str__` method that displays what error code it is.

Ex: IOError
```
[Errno 2] No such file or directory: 'foo.txt'
<type 'exceptions.IOError'>
```